detectron2/data/detection_utils.py

```diff
@@ -97,6 +100,51 @@ def convert_image_to_rgb(image, format):
     return image
 
 
+def apply_exif_orientation(image):
```

Comment:
prefix it by `_` since we don't expect it to be used outside of this file

Reply:
Good call